telemetry/client.go

```diff
@@ -96,5 +233,7 @@ func (c *Client) UpdateEnvelopeProcessingError(shhMessage *types.Message, proces
 	_, err := c.httpClient.Post(url, "application/json", bytes.NewBuffer(body))
 	if err != nil {
 		c.logger.Error("Error sending envelope update to telemetry server", zap.Error(err))
+	} else {
+		c.logger.Debug("Successfully pushed envelope processing error to telemetry server", zap.String("hash", types.EncodeHex(shhMessage.Hash)))
```

Comment:
Might be too noisy due to it being the expected result. Consider not logging this

Reply:
removed in latest commit